src/ImageService/Controllers/ImageNewsController.cs

```diff
@@ -1,21 +1,41 @@
-﻿using LT.DigitalOffice.ImageService.Business.Commands.ImageNews.Interfaces;
+﻿using System;
+using System.Collections.Generic;
+using System.Net;
+using LT.DigitalOffice.ImageService.Business.Commands.ImageNews.Interfaces;
+using LT.DigitalOffice.ImageService.Models.Dto.Requests;
 using LT.DigitalOffice.ImageService.Models.Dto.Responses;
+using LT.DigitalOffice.Kernel.Enums;
 using LT.DigitalOffice.Kernel.Responses;
+using Microsoft.AspNetCore.Http;
 using Microsoft.AspNetCore.Mvc;
-using System;
 
 namespace LT.DigitalOffice.ImageService.Controllers
 {
-    [ApiController]
-    [Route("[controller]")]
-    public class ImageNewsController
+  [ApiController]
+  [Route("[controller]")]
+  public class ImageNewsController : ControllerBase
+  {
+    private readonly IHttpContextAccessor _httpContextAccessor;
```

Comment:
for what do you use the accessor here?

Reply:
There was a check, but we removed it, forgot to remove accessor.
Done